pkgs/by-name/ca/capypdf/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  freetype,
+  lcms2,
+  libjpeg,
+  libpng,
+  libtiff,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "capypdf";
+  version = "0.14.0";
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "jpakkane";
+    repo = "capypdf";
+    rev = finalAttrs.version;
+    hash = "sha256-izc5EReAeDpR4Urktii5kJIZai69ga4QweSbwzuLNxc=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    freetype
+    lcms2
+    libpng
+    libjpeg
+    libtiff
+    zlib
+  ];
+
+  meta = {
+    description = "A fully color managed PDF generation library";
+    homepage = "https://github.com/jpakkane/capypdf";
```

Comment:
```suggestion
    inherit (finalAttrs.src.meta) homepage;
```

Reply:
I prefer the former, as it allows opening the homepage directly from GitHub or text editor.